src/transformers/visitors/toContractVisitor.ts

```diff
@@ -425,19 +458,11 @@ export default {
       let newNode: any;
 
       // If this node is a require statement, it might include arguments which themselves are expressions which need to be traversed. So rather than build a corresponding 'assert' node upon entry, we'll first traverse into the arguments, build their nodes, and then upon _exit_ build the assert node.
-
       if (path.isRequireStatement()) {
         // If the 'require' statement contains secret state variables, we'll presume the circuit will perform that logic, so we'll do nothing in the contract.
-        const findSecretSubnode = (p: any, state: any) => {
-          const isSecret = p.getReferencedNode()?.isSecret;
-
-          if (isSecret) {
-            state.secretFound = true;
-          }
-        };
-        const subState = { secretFound: false };
-        path.traversePathsFast(findSecretSubnode, subState);
-        if (subState.secretFound) {
+        const subState = { interactsWithSecret: false };
+        path.traversePathsFast(interactsWithSecretVisitor, subState);
+        if (subState.interactsWithSecret) {
```

Comment:
Change to `interactsWithSecret` is just for cleanliness + consistency (we use the same method in `toOrchestration`)